src/ui/common/src/components/workflows/ReactFlowCanvas.tsx

```diff
@@ -33,6 +33,7 @@ const ReactFlowCanvas: React.FC<ReactFlowCanvasProps> = ({
     (state: RootState) => state.openSideSheetReducer
   );
   const { fitView } = useReactFlow();
+  useEffect(() => { fitView() }, []);
```

Comment:
TODO: Will need to bump version in package.json and re-publish before merging this PR.

Reply:
NOTE: Will bump after #42 is merged so we can just do it all in one go.